<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParkingAccessQuery</class>
 <widget class="QWidget" name="ParkingAccessQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1360</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1360</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="le_plate">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>60</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="labe_location">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>65</y>
     <width>180</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>车牌号：</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>206</width>
     <height>37</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_type">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>类型：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_type">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>进入</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>离开</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_startDate">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>180</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开始日期：</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_start">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>175</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_endDate">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>180</y>
     <width>180</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>结束日期：</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_end">
   <property name="geometry">
    <rect>
     <x>584</x>
     <y>175</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_query">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>140</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>筛选</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
